apps/web/src/app/archive/wish/layout.tsx

```
@@ -26,12 +25,8 @@ async function ArchiveLayout({ children }: ArchiveLayoutProps) {
       queryKey: ['me'],
       queryFn: getMe,
     });
-    if (user.identityType !== 'MEMBER') {
-      /** 위시 페이지는 로그인 유도 화면을 렌더 */
-      if (redirectPath?.startsWith(ROUTES.WISHLIST)) return <WishLoginRequired />;
-
-      redirect(getLoginPath(redirectPath));
-    }
+    /** 위시 페이지는 멤버가 아니면 로그인 유도 화면을 렌더 */
+    if (user.identityType !== 'MEMBER') return <WishLoginRequired />;
```

ESLint

[error] Error: Avoid constructing JSX within try/catch

React does not immediately render components when JSX is rendered, so any errors from this component will not be caught by the try/catch. To catch errors in rendering a given component, wrap that component in an error boundary. (https://react.dev/reference/react/Component#catching-rendering-errors-with-an-error-boundary).

/home/jailuser/git/apps/web/src/app/archive/wish/layout.tsx:29:48
  27 |     });
  28 |     /** 위시 페이지는 멤버가 아니면 로그인 유도 화면을 렌더 */
> 29 |     if (user.identityType !== 'MEMBER') return <WishLoginRequired />;
     |                                                ^^^^^^^^^^^^^^^^^^^^^ Avoid constructing JSX within try/catch
  30 |   } catch (error) {
  31 |     if (!isAxiosError<ApiErrorResponseT>(error)) throw error;
  32 |

(react-hooks/error-boundaries)